src/Blueprint/Blueprint/Properties/Resources.resx

```diff
@@ -0,0 +1,147 @@
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AssignmentExists" xml:space="preserve">
+    <value>An assignment with name '{0}' in Subscription '{1}' already exists. Please use Set-AzBlueprintAssignment to update an existing assignment.</value>
+  </data>
+  <data name="AssignmentNotExist" xml:space="preserve">
+    <value>An assignment with name '{0}' in Subscription '{1}' does not exists. Please use New-AzBlueprintAssignment to create a new assignment.</value>
+  </data>
+  <data name="BlueprintAssignmentNotFound" xml:space="preserve">
+    <value>Blueprint assignment '{0}' could not be found within Subscription '{1}'.</value>
+  </data>
+  <data name="CreateAssignmentShouldProcessString" xml:space="preserve">
+    <value>Creating blueprint assignment '{0}'</value>
+  </data>
+  <data name="DeleteAssignmentShouldProcessString" xml:space="preserve">
+    <value>Deleting blueprint assignment '{0}' in Subscription '{1}'</value>
+  </data>
+  <data name="ManagementGroupNotFound" xml:space="preserve">
+    <value>We can't find any Management Groups that you have access to.</value>
+  </data>
+  <data name="NoBlueprintAssignmentsNotFound" xml:space="preserve">
+    <value>Could not find any Blueprints assignments within '{0}'. Please check the Subscription Id and try again.</value>
+  </data>
+  <data name="NoBlueprintsFound" xml:space="preserve">
+    <value>Could not find any Blueprints.</value>
+  </data>
+  <data name="PublishedBlueprintNotFound" xml:space="preserve">
```

Comment:
>PublishedBlueprintNotFound [](start = 14, length = 26)

Do we need a separate message if they're looking for a specific version?

Reply:
These are the artifacts from the time I was trying to produce proper error messaging. Then decided to relay the error our API sent. So, removing them from the .resx file.

---
In reply to: [257123122](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257123122) [](ancestors = 257123122)